### PR TITLE
(Closes #27) Implement the custom prompt string

### LINKED ISSUE #27: Prompt string
My current leaning:
`[<filename>: <current directory>] > `

so if you were in `foo.h5` and within the `bar` group, it would appear as `[foo.h5: bar] > `

I prefer just the group name (keeps the prompt from growing far to long), not the full path, though perhaps that should be configurable

if in root, shows up as `/`

Not sold on the last character

### PR DESCRIPTION
**Pull request by ksunden**
shows up as `[foo.h5: bar/] > ` where foo.h5 is the filename and
bar is the group of the current working directory

In implementing I added \_\_init__ with a one other slight modification
to default behaviour, I have turned abbreviations on, which allows
non-ambiguous commands to be run with the shortest such name

e.g. `history` can be `hist` or `hi` but not `h` because `help` makes
that ambiguous

### CODE REVIEW

File: src/cli.py [hunk 1]
```diff
@@ -6,14 +6,25 @@
...
 
     def precmd(self, line):
         if not line.startswith('load') and \
            (line.endswith('.h5') or line.endswith('.hdf5')):
             line = 'load ' + line
         return line
 
+    def postcmd(self, stop, line):
+        if self.explorer:
+            self.prompt = '[{}: {}/] > '.format(
```
Comment (lphk92) [inline src/cli.py:23]: Add a beginning slash to the prompt so it looks like [{}: /{}/] >

**Review by lphk92 — CHANGES_REQUESTED** [review]

File: src/cli.py (hunk 1 above)
Comment (ksunden) [reply src/cli.py:23]: Why would I do that? That would make it appear as if it is a root relative path

Also, the current functionality works for root as well as directories, that would make it appear as [foo.h5: //] > when in root

Comment (lphk92) [reply src/cli.py:23]: Shouldn't it be a root relative path? If it's not, then what path are you showing?

Comment (ksunden) [reply src/cli.py:23]: It is simply the basename of the directory you are currently in

my Bash prompt, for the record, is similar [user@host directory]$, not shown relative to ~ or /, just the name of the directory

Comment (ksunden) [reply src/cli.py:23]: This may be personal preference, but I don't like my prompt taking up half the line

Comment (lphk92) [reply src/cli.py:23]: Ahh gotcha, yea that makes sense

**Pull request merged**